.drone.yml

```
@@ -5014,11 +5014,12 @@ steps:
       - |
         # write config files
         mkdir -p /go/reprepro/teleport/conf /go/reprepro/teleport/public
+        # we have "arm" packages even though that's not a real debian architecture
```

This comment tells future maintainer what, but not why.  To inject a "why", maybe something like:

```suggestion
        # we offer "arm" packages for backwards compatibility even though that's not a real debian architecture
```

---

69b156859909a068620c70e73e2c55b41aaa7cb8